tests/testthat/test-verb-select.R

```diff
@@ -71,6 +71,24 @@ test_that("select preserves grouping vars", {
   expect_named(out, c("b", "a"))
 })
 
+test_that("select handles order vars", {
+  lf <- lazy_frame(x = 1, y = 1, z = 1)
+  # can drop order vars
+  expect_equal(lf %>% window_order(y) %>% select(-y) %>% op_sort(), list())
+  expect_equal(lf %>% window_order(desc(y)) %>% select(-y) %>% op_sort(), list())
+  # can rename order vars
+  expect_equal(lf %>% window_order(y) %>% select(y2 = y) %>% op_sort(), list(expr(y2)))
+  expect_equal(
+    lf %>% window_order(desc(y)) %>% select(y2 = y) %>% op_sort(),
+    list(expr(desc(y2)))
+  )
+  # keeps sort order
```

Comment:
I think the order of variables shouldn't be touched by `select()`

Reply:
Agreed